lib/options-handler.ts

```diff
@@ -56,6 +56,7 @@ export type VersionInfo = {
     options: string[];
     hidden: boolean;
     packagedheaders?: boolean;
+    $order?: number;
```

Comment:
This is used in https://github.com/compiler-explorer/compiler-explorer/blob/9e050befc6025e4bc37a4a95680cf6bc03c1fe8c/lib/options-handler.ts#L369 , should be a member of VersionInfo.